### PR TITLE
Update comparison doc

### PR DESCRIPTION
**Pull request by Nirothipan**
$subject

### CODE REVIEW

File: en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md [hunk 1]
```diff
@@ -218,39 +218,6 @@ The following features, which are not needed for MSA-based deployments or not us
...
-		<td>
-			Coordination support for message processors, inbound endpoints, and tasks, which is built into the ESB profile, is removed from the Micro Integrator. This feature is not a requirement in the microservices world.</br></br> However, coordination support will be added to the Micro Integrator in a future version to support standard, centralized deployment architectures.
-		</td>
-		<td>
-			-
-		</td>
-	</tr>
 	<tr>
 		<td>
 			Database-based Registry
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md:223]: RDBMS-based Registry?

Comment (Nirothipan) [reply en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md:223]: no , MI is file based as mentioned in alternative.

File: en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md [hunk 2]
```diff
@@ -9,7 +9,8 @@
  
 This [WSO2 EI 6.x product family](http://docs.wso2.com/enterprise-integrator) offers the conventional, centralized integration solution of the WSO2 middleware stack. WSO2 EI 6.x includes the ESB profile and other supporting profiles packaged in a single distribution.
  
-The new [WSO2 Enterprise Integrator (WSO2 EI 7.0.0)](https://ei.docs.wso2.com/en/latest/) is a hybrid platform that enables API-centric integration supporting various integration architecture styles: microservices architecture, cloud-native architecture, or a centralized ESB architecture. This integration platform offers a graphical/configuration-driven approach to developing integrations for any of the architectural styles.
+The new [WSO2 Enterprise Integrator (WSO2 EI 7.x)](https://ei.docs.wso2.com/en/latest/) is a hybrid platform that
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md:12]: Shall we point to a product page here or remove this link? Pointing the user to the doc landing page when they are in the docs already seems pointless.

File: en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md:223]: Yes.. I mean.. shall we reword 'Database-based Registry' to 'RDBMS-based Registry'

File: en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md (hunk 2 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/references/comparisong-mi7-ei6xx.md:12]: +1 to remove.

**Pull request merged**